internal/state/change_processor_test.go

```diff
@@ -1995,13 +1995,15 @@ var _ = Describe("ChangeProcessor", func() {
 		})
 
 		assertHREvent := func() {
-			e := <-fakeEventRecorder.Events
+			var e string
+			EventuallyWithOffset(1, fakeEventRecorder.Events).Should(Receive(&e))
```

Comment:
Why change this to eventually? I thought we were trying to avoid polling in unit tests?

Reply:
in case there is a bug in the code, and the channel doesn't receive anything, the previous version would just hang (meaning the test run would hang). So I added eventually to prevent that.